woocommerce/woocommerce
# [LYS] Remove Coming Soon page and id

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





https://github.com/woocommerce/woocommerce/pull/46508 Applies a template only method of rendering a Coming Soon page. This makes the Coming Soon page that is created on install/update no longer relevant. This PR removes it along with the associated option `woocommerce_coming_soon_page_id`.

### Items to Note

* This PR does not handle updating the Coming Soon edit link, that will be handled separately in https://github.com/woocommerce/woocommerce/issues/46562
* `woocommerce_coming_soon_page_id` has been removed
* `ComingSoonCacheInvalidator` logic has been removed



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure LYS is still functional by spinning up a JN site
2. Visit any front end page in incognito mode, see the entire site coming soon page rendered
3. Change the visibility settings to `Restrict to store pages only` 
4. Now visit any store page and see the coming soon store only page. 
5. Note the Coming Soon page is no longer in the header



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Edits previously unreleased feature

</details>
